x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/MetadataAttribute.java

```diff
@@ -49,7 +50,9 @@ public class MetadataAttribute extends TypedAttribute {
         SourceFieldMapper.NAME,
         tuple(DataType.SOURCE, false),
         IndexModeFieldMapper.NAME,
-        tuple(DataType.KEYWORD, true)
+        tuple(DataType.KEYWORD, true),
+        SCORE,
```

Comment:
I assume we want the `SCORE` metadata field available for snapshots only at this point?

Reply:
Done [7bee884](https://github.com/elastic/elasticsearch/pull/113120/commits/7bee88403f622c349fc8b537d3a5a94c7bf02df2)